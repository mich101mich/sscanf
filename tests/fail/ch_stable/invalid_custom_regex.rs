fn main() {
    sscanf::scanf!("", "   {/ab(ab/}   ", String).unwrap();
}
