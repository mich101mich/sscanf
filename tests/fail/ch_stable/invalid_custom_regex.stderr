error: regex parse error:
           ab(ab
             ^
       error: unclosed group

       In custom Regex format option:
       At "   {:/ab(ab/}   "
                 ^^^^^
 --> tests/fail/ch_stable/invalid_custom_regex.rs:2:24
  |
2 |     sscanf::scanf!("", "   {:/ab(ab/}   ", String).unwrap();
  |                        ^^^^^^^^^^^^^^^^^^
